### PR TITLE
Jwineman/prettier

### PR DESCRIPTION
**Pull request by jwineman**
I want to try out [`prettier/prettier`](https://github.com/prettier/prettier/) since our current eslint config isn't very strict.

- Modified .eslintrc to use `eslint-plugin-prettier`
- Modifed `npm run format` to use `prettier/prettier`.

Prettier has editor configs for [atom](https://atom.io/packages/prettier-atom)/[sublime](https://packagecontrol.io/packages/JsPrettier) etc which will auto format on save.

### CODE REVIEW

**Comment by jwineman** [comment]
CC @tajo

**Comment by tajo** [comment]
LGTM and apply on all JS projects.

File: src/actions/activeZone.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
-import * as ActionTypes from '../constants/ActionTypes';
-import { asyncDNSRecordFetchList } from './zoneDnsRecords';
-import { asyncZoneFetchAnalytics } from './zoneAnalytics';
-import { asyncZoneRailgunFetchAll } from './zoneRailgun';
-import { asyncZoneFetchSettings } from './zoneSettings';
-import { asyncPluginFetchSettings } from './pluginSettings';
```
Comment (thellimist) [inline src/actions/activeZone.js]: Do we want `singleQuote: false`? In internal repo we're using `singleQuote: true`

Comment (jwineman) [inline src/actions/activeZone.js]: If cf-www-next ends up with `singleQuote: true` I'll go back and update it again.  Just wanted to get this PR out before the UI meeting so we had an example to talk about.

**Pull request merged**